hdinsight-new-datalake-store-azure-storage/azuredeploy.json

```diff
@@ -0,0 +1,206 @@
+			}
+		},
+		"sshPassword": {
+			"type": "securestring",
+			"metadata": {
+				"description": "The password must be at least 10 characters in length and must contain at least one digit, one non-alphanumeric character, and one upper or lower case letter."
+			}
+		},
+		"clusterStorageAccountName": {
+			"type": "string",
+			"metadata": {
+				"description": "The name of the Azure storage account to be created and be used as the cluster's primary storage."
+			}
+		},
+        "adlStoreName": {
+			"type": "string",
+			"metadata": {
+				"description": "The name of the Azure Data Lake Store account to be created and connected to the cluster."
+			}
+		},
+        "aadTenantId": {
+            "type": "string",
+            "metadata": {
+				"description": "The tenant ID (guid) of the Azure Active Directory (AAD) tenant where the service principal resides."
+			}
+        },
+        "servicePrincipalObjectId": {
+            "type": "string",
+            "metadata": {
+				"description": "The AAD object ID (guid) of the service principal that represents the HDInsight cluster. The service principal will be given permissions on the root folder of the Data Lake Store account."
+			}
+        },
+        "servicePrincipalApplicationId": {
+            "type": "string",
+            "metadata": {
+				"description": "The AAD application ID (guid) of the service principal that represents the HDInsight cluster. The service principal will be given permissions on the root folder of the Data Lake Store account."
+			}
+        },
+        "servicePrincipalCertificateContents": {
+            "type": "securestring",
+            "metadata": {
+				"description": "The base-64-encoded contents of the PFX certificate file that can be used to authenticate as the service principal that represents the HDInsight cluster."
+			}
+        },
+        "servicePrincipalCertificatePassword": {
+            "type": "securestring",
+            "metadata": {
+				"description": "The password securing the PFX certificate file that can be used to authenticate as the service principal that represents the HDInsight cluster."
+			}
+        },
+		"clusterWorkerNodeCount": {
+			"type": "int",
+			"defaultValue": 4,
+			"metadata": {
+				"description": "The number of nodes in the HDInsight cluster."
+			}
+		}
+	},
+	"variables": {
+		"defaultApiVersion": "2015-06-15",
+		"clusterApiVersion": "2015-03-01-preview",
+        "adlsApiVersion": "2015-10-01-preview"
+	},
+	"resources": [{
+        "name": "[parameters('adlStoreName')]",
+        "type": "Microsoft.DataLakeStore/accounts",
+        "location": "East US 2",
+        "apiVersion": "[variables('adlsApiVersion')]",
+        "dependsOn": [],
+        "tags": { },
+        "properties": {
+            "initialUser": "[parameters('servicePrincipalObjectId')]"
+        }
+    },
+    {
+		"name": "[parameters('clusterStorageAccountName')]",
+		"type": "Microsoft.Storage/storageAccounts",
+		"location": "East US 2",
```

Comment:
Today, the location needs to be "East US 2" for all resources in a deployment that includes a Data Lake resource.

Data Lake (Store and Analytics) is currently available only in East US 2.